### ISSUE TITLE
EDAM.obo and EDAM.owl mismatch 

### ISSUE THREAD

**Issue by EricDeveaud** [edamontology/edamontology#31]
while  working on EDAM.owl and EDAM.obo downloaded from http://edamontology.org/EDAM.owl and 
http://edamontology.org/EDAM.obo
I noted some mismatch beetween 2 files regarding the same term.

eg topic_0100 from owl definitions is tagged deprecated 
```
    <!-- http://edamontology.org/topic_0100 -->

    <owl:Class rdf:about="http://edamontology.org/topic_0100">
        <rdfs:label>Nucleic acid restriction</rdfs:label>
        <rdfs:subClassOf rdf:resource="&oboInOwl;ObsoleteClass"/>
        <obsolete_since>1.3</obsolete_since>
        <created_in>beta12orEarlier</created_in>
        <oboInOwl:hasDefinition>Topic for the study of restriction enzymes, their cleavage sites and the restriction of nucleic acids.</oboInOwl:hasDefinition>
        <owl:deprecated>true</owl:deprecated>
        <oboInOwl:consider rdf:resource="http://edamontology.org/topic_0747"/>
        <oboInOwl:inSubset rdf:resource="&oboOther;edam#obsolete"/>
    </owl:Class>
```

while obo definition is not:

```
[Term]
id: EDAM_topic:0100
name: Nucleic acid restriction
subset: bioinformatics
subset: edam
subset: topics
created_in: "beta12orEarlier"
def: "Topic for the study of restriction enzymes, their cleavage sites and the restriction of nucleic acids." [http://edamontology.org]
namespace: topic
is_a: EDAM_topic:0640 ! Nucleic acid sequence analysis
is_a: EDAM_topic:0821 ! Enzymes and reactions
```
and so does not have consider as in owl.

NB: same problem with topic_0748, topic_0179, operation_2448

Eric

**Comment by jongithub** [comment]
Hi Eric

We no longer support EDAM in OBO format and that file you're looking at is for a rather old version.  Sorry about it.  If we had more resources we'd like to support OBO, but can't right now.

Cheers 

Jon

**Comment by EricDeveaud** [comment]
Jon, 

if obo format is no longer supported, why not remove it or clearly tag it as deprecated ? 
I must admit that I did not read the homepage where it is stated that 
```
Please note that the last stable version of EDAM available in OBO format was version 1.2. Because the conversion from OWL to OBO hasn't yet fully been automated for EDAM, we will only resume providing OBO format in case of substantial demand or full automation of the conversion.
```

having the EDAM.owl or EDAM.obo fiels named according to their version would avoid such confusion.

**Comment by hmenager** [comment]
The issue is not what was announced initially then. I will close this issue, and create another one to record the OBO file release issue
